### DISCUSSION TITLE
How can i disable the color scheme?

### DISCUSSION THREAD

**Discussion by zelosleone** [LazyVim/LazyVim#1465] [Q&A] [answered]
How can i disable the color scheme so acrylic effects on terminal shows? I tried the default color scheme via: it works; however, it is not consistent, and it throws errors.

**Comment by dpetka2001** [comment, marked as answer]
You can do 
```lua
  { "LazyVim/LazyVim", opts = {
    colorscheme = "darkblue",
  } },
```
in your config to change the default colorscheme to whatever you'd like.

**Reply by zelosleone** [reply]
Perfect, thank you very much!
